.circleci/config.yml

```diff
@@ -88,6 +87,13 @@ workflows:
           filters:
             tags:
               # only upload the docker container on semver tags
-              only: /[0-9]\.[0-9]+\.[0-9]+/
+              only: /^v[0-9]\.[0-9]+\.[0-9]+/
             branches:
-              only: master
+              # filters appear to be OR'd together, so we can't specify just
+              # 'master' (as it tries to upload with every commit to
+              # master).
```

Comment:
we usually deploy master as the `latest` tag

Reply:
I'm not sure I follow this, so likely should be a separate bug. As configured now, an image is pushed to dockerhub whenever a tag matching a 3 level semver tag is applied (with a leading 'v'). It does not matter which branch the tag is applied to.